requirements.txt

```diff
@@ -2,3 +2,5 @@ django==2.2.24
 django-authtools>=1.6.0
 django-widget-tweaks>=1.4.3
 psycopg2>=2.7.6
+pandas>=1.4.0
+lockfile>=0.12.2
```

Comment:
`lockfile` is deprecated and not under development since 2015. Perhaps you can use [fasteners](https://pypi.org/project/fasteners/).